### PR TITLE
Add missing imports to the integration tests

### PR DESCRIPTION
**Pull request by khairalanam**
This PR adds missing imports to many of the integration tests.

The reference tests are updated successfully without any errors.

### CODE REVIEW

File: tests/reference/runtime-test_list_01-3ee9b3e.stdout [hunk 1]
```diff
@@ -0,0 +1 @@
+The command 'link /NOLOGO /OUT:test_list_01.out test_list_01.out.tmp.o $DIR\src\bin/../runtime\lpython_runtime_static.lib > NUL' failed.
```
Comment (Thirumalai-Shaktivel) [inline tests/reference/runtime-test_list_01-3ee9b3e.stdout:1]: I think these changes has to be removed!

**Comment by Thirumalai-Shaktivel** [comment]
@khairalanam for updating tests in Windows, please do: `python run_tests.py -u --skip-run-with-dbg`

**Comment by khairalanam** [comment]
> @khairalanam for updating tests in Windows, please do: `python run_tests.py -u --skip-run-with-dbg`

I ran this command for updating the tests for this PR.

**Comment by khairalanam** [comment]
Should I run the command again?

**Comment by Thirumalai-Shaktivel** [comment]
I see, there seems to be some problem in the Windows, we have to debug that!
Do this, just to revert the `runtime-test_list_01_*.stdout` and `runtime-test_list_02_*.stdout` changes. Then I think this PR is good to go in.

**Comment by khairalanam** [comment]
> I see, there seems to be some problem in the Windows, we have to debug that! Do this, just to revert the `runtime-test_list_01_*.stdout` and `runtime-test_list_02_*.stdout` changes. Then I think this PR is good to go in.

Can this be done manually?

File: tests/reference/runtime-test_list_01-3ee9b3e.stdout (hunk 1 above)
Comment (czgdp1807) [reply tests/reference/runtime-test_list_01-3ee9b3e.stdout]: These files were unused anyways I guess. Check https://github.com/lcompilers/lpython/pull/1655/commits/45973741a5ee1a5db7a96525099b7ce4d3769150

**Pull request merged**